TweetMate/src/main/java/com/ybsystem/tweetmate/models/entities/twitter/TwitterStatus.java

```diff
@@ -239,12 +242,12 @@ private static String createTweetText(Status status) {
     private static String createRelativeTime(Status status) {
         // Convert
         Date date = status.getCreatedAt();
-        String text = new TimeSpanConverter().toTimeSpanString(date);
+        String text = new TwitterTimeSpanConverter().toTimeSpanString(date);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "text". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1488)